src/ip.py

```diff
@@ -32,6 +34,9 @@
 ARP = "/usr/sbin/arp"
 NETWORKSETUP = "/usr/sbin/networksetup"
 
+# Globals
+json_print = False
```

Comment:
Could you please rework that into a method argument similar to `af` ? I would prefer to avoid global state.

Reply:
Yes, sure, as you prefer.